contrib/installer/linux/io.github.mgerhardy.vengi.metainfo.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppdataEdit 5.0-->
+  <id>io.github.mgerhardy.vengi</id>
+  <name>vengi</name>
```

Comment:
is the id correct? shouldn't this this be more like io.github.mgerhardy.vengi.voxedit to be able to support e.g. io.github.mgerhardy.vengi.voxconvert or io.github.mgerhardy.vengi.thumbnailer (and others in the future)?

Reply:
Renamed it. Command line tools are not allowed on @flathub though.